### ISSUE TITLE
Multiple processes in the GUI

### ISSUE THREAD

**Issue by cprando** [veraPDF/veraPDF-library#999]
When I run a validation I have to wait until the process ends before another task can be launched. Is it possible to run it in the background (e.g. showing a status bas at the bottom of the GUI as in DPF Manager and MediaConch) and launch several tasks in parallel, giving the possibility to view/download the corresponding report once they are finished?
